learn/getting_started/installation.md

```diff
@@ -47,6 +47,7 @@ docker run -it --rm \
 ```
 
 Data written to a **Docker container is not persistent** and is deleted along with the container when the latter is stopped. Docker volumes are not deleted when containers are removed. It is then recommended to share volumes between your containers and your host machine to provide persistent storage. MeiliSearch writes data to `/data.ms`
+If you are new to Docker, you can learn more about it on the [official documentation](https://docs.docker.com/get-docker/).
```

Comment:
Really not a fan of that line break that doesn't start a new line. The division of the content becomes visually unclear, IMHO, and I think we need to either integrate into the previous paragraph or push it down one line:

![Screen Shot 2021-09-30 at 16 24 35](https://user-images.githubusercontent.com/1923183/135473908-a87e5432-66ac-4e81-88ca-802182d6a82c.jpg)
 
Also, simplifying the sentence a bit:
```suggestion

You can learn more about Docker on the [official documentation](https://docs.docker.com/get-docker/).
```